bigtable-hbase-2.x-replication/src/main/java/com/google/cloud/bigtable/hbase/replication/HBaseMetricsExporter.java

```diff
@@ -0,0 +1,20 @@
+package com.google.cloud.bigtable.hbase.replication;
+
+import com.google.cloud.bigtable.hbase.replication.metrics.MetricsExporter;
+import org.apache.hadoop.hbase.replication.regionserver.MetricsSource;
+/**
+ * HBaseMetricsExporter implements MetricExporter which bridges with MetricsSource.
+ */
+public class HBaseMetricsExporter implements MetricsExporter {
+    // set this as static
```

Comment:
Usually, its much more valuable to comment on "why" you kept it static. Describing the code usually not helpful.